elegant-progressbar-for-a-more-civilized-ageConfigVersion.cmake

```diff
@@ -0,0 +1,26 @@
+file(STRINGS "${CMAKE_CURRENT_LIST_DIR}/include/elegantProgressbars/version.hpp" elegantProgressbars_VERSION_MAJOR_HPP REGEX "#define ELEGANTPROGRESSBARS_VERSION_MAJOR ")
+file(STRINGS "${CMAKE_CURRENT_LIST_DIR}/include/elegantProgressbars/version.hpp"  elegantProgressbars_VERSION_MINOR_HPP REGEX "#define ELEGANTPROGRESSBARS_VERSION_MINOR ")
+file(STRINGS "${CMAKE_CURRENT_LIST_DIR}/include/elegantProgressbars/version.hpp"  elegantProgressbars_VERSION_PATCH_HPP REGEX "#define ELEGANTPROGRESSBARS_VERSION_PATCH ")
```

Comment:
include_directories("${CMAKE_CURRENT_LIST_DIR}/../include......."), if the cmake config is located in its own cmake